src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_quantization.cpp

```diff
@@ -588,6 +588,10 @@ void prepare_quantization::prepare_asymmetric_quantization(program &p, convoluti
         in1.get_output_layout().data_type != data_types::i8)
         return;
 
+    if (asymmetric_data && in0.is_dynamic() &&
+        in0.get_output_layout().get_partial_shape()[1].get_interval().size() != 1)
```

Comment:
1. Please add a constant like `const size_t feature_idx = 1;` and use it here and below.
2. `in0.get_output_layout().get_partial_shape()[1].get_interval().size() != 1` seems to be not needed if you have `in0.is_dynamic()`. So I think you need to keep only one of those conditions (probably it can work if feature dim is static, but others are dynamic)

Reply:
Agree, I applied to check 'get_partial_shape()[feature_idx].is_dynamic()'. LCM is static shape in feature and others are dynamic.